### ISSUE TITLE
Question . Is it possible to report an issue or PR to EE version ?

### ISSUE THREAD

**Issue by cherednichenkoa** [magento/magento2#16696]
Hello, during Magento upgrade from EE 2.2.3 -> EE 2.2.5 I have noticed that several extensions, are not compatible with magento split database, I see it is 3rd party vendor
Modules below are not compatible with split database solution, I understand that modules from 3rd party vendors, but how is it possible that special feature of EE, do not assume compatibility with EE features ?

1. Vertex_Tax 
2. Amazon_Core, Amazon_Payment

All those extensions does not even assume that any connection to sales or checkout database exists

**Comment by magento-engcom-team** [comment]
Hi @cherednichenkoa. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation.

@cherednichenkoa do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by orlangur** [comment]
Hi @cherednichenkoa EE (currently known as Magento Commerce) issues are supposed to be reported via 
> Please report Magento Commerce issues via the Support portal of your account or Partner portal if you are a partner reporting on behalf of a merchant.
GitHub is intended for Magento Open Source bug reports given no account management for Magento Open Source users. This will allow for proper tracking of issues at the account level.

There is a partner program which allows contributions for ee/b2b code.

As a workaround you can get rid of unneeded modules with
> FYI if you are using `composer.json` then just exclude the module if you don't need or use the service, i.e. after "require-dev" section 
```
    "replace": {
        "dotmailer/dotmailer-magento2-extension": "*",
        "amzn/amazon-pay-and-login-magento-2-module": "*",
        "klarna/module-core": "*",
        "klarna/module-kp": "*",
        "klarna/module-ordermanagement": "*",
        "shopialfb/facebook-module": "*",
        "temando/module-shipping-m2": "*",
        "vertex/module-tax": "*",
        "magento/module-signifyd": "*"
    },
``` 
> This will remove after `composer update` those.

**Comment by engcom-backlog-nickolas** [comment]
Hello @cherednichenkoa, I'm closing this issue, as previous comment, I believe, answer your question completely. And as being said , GitHub is intended for Magento Open Source bug reports.

Best regards!

**Comment by navarr** [comment]
@cherednichenkoa Hi!

What issues are you having with Vertex in a split-database solution?

**Comment by cherednichenkoa** [comment]
@navarr , vertex does not assume split database feature in Schema scripts, that should affect sales tables 
at least 2.1.8 version does it

**Comment by cherednichenkoa** [comment]
@navarr , it takes default connection instead of sales connection in schema scripts that update sales tables (tables that are not available via default connection, and located in separate database), so it gives fatal errors during setup upgrade

**Comment by navarr** [comment]
@cherednichenkoa I think I see where the problem can exist.  This is when installing Vertex on pre-existing split-database installs, correct?

If you have a log readily available I'd love to have it for reference - if not no worries, I think I know the issues and will try to resolve in a future version.

**Comment by navarr** [comment]
@cherednichenkoa Thanks for your report.  I've created a fix.  It should be available no later than Magento 2.3 / Magento 2.2.7.
